internal/cloud/backend_test.go

```diff
@@ -269,6 +289,49 @@ func TestCloud_configWithEnvVars(t *testing.T) {
 			},
 			expectedHostname: "app.terraform.io",
 		},
+		"null workspaces": {
+			config: cty.ObjectVal(map[string]cty.Value{
+				"hostname":     cty.StringVal("app.terraform.io"),
+				"token":        cty.NullVal(cty.String),
+				"organization": cty.StringVal("hashicorp"),
+				"workspaces":   cty.NullVal(cty.String),
```

Comment:
As I understand it, this (and the other `NullVal(String)` value below) is not realistic data to give as input to this function, and while it currently works I think it's misleading. I think a more correct value here would be:

```suggestion
				"workspaces":   cty.NullVal(cty.Object(map[string]cty.Type{"name": cty.String, "tags": cty.Set(cty.String)})),
```